frontend/src/components/episode/components/comments-list/comments-list.tsx

```diff
@@ -0,0 +1,26 @@
+import { Comment as TComment } from 'common/types/types';
+import styles from './styles.module.scss';
+import Comment from '../comment/comment';
```

Comment:
Comment should be at least on the same level as comments-list component

Reply:
What do you mean? how should it be? Now it look like that
\episode
  \components
    \comment
        comment.tsx
        styles.module.scss
    \comments-list
        comments-list.tsx
        styles.module.scss